go/updater/updater.go

```diff
@@ -101,6 +101,18 @@ func (u *Updater) checkForUpdate(skipAssetDownload bool, force bool, requested b
 		return
 	}
 
+	// do only if instructions are empty/nil
+	if update.Instructions == nil || *update.Instructions == "" {
+		instructions, err := libkb.PlatformSpecificUpgradeInstructionsString()
+		u.log.Info("Updating instructions! %s, err: %s", instructions, err)
```

Comment:
Can we clean up the logging here make this a debug and move the err to the if err != nil part?